src/antsibull_build/build_ansible_commands.py

```diff
@@ -435,6 +437,57 @@ def validate_deps_data(
                 )
 
 
+def check_for_regressions(
+    ansible_version: PypiVer, deps: DependencyFileData, data_dir: os.PathLike[str] | str
+) -> None:
+    dependencies = {
+        PypiVer(deps_ansible_version): deps
+        for deps_ansible_version, deps in load_all_dependency_files(
+            data_dir,
+            accept_deps_file=lambda path, deps_ansible_version: PypiVer(
+                deps_ansible_version
+            )
```

Comment:
This lambda is wrapped in a way that makes it hard to understand, but I guess that's black's fault. Meh.